src/AppInstallerRepositoryCore/Rest/RestClient.cpp

```diff
@@ -80,13 +80,17 @@ namespace AppInstaller::Repository::Rest
         return *commonVersions.rbegin();
     }
 
-    std::unique_ptr<Schema::IRestClient> RestClient::GetSupportedInterface(const std::string& api, const Version& version)
+    std::unique_ptr<Schema::IRestClient> RestClient::GetSupportedInterface(const std::string& api, const IRestClient::Information& information, const Version& version)
     {
         if (version == Version_1_0_0)
         {
             return std::make_unique<Schema::V1_0::Interface>(api);
         }
-       
+        else if (version == Version_1_1_0)

```

Comment:
>version == Version_1_1_0) [](http://example.com/codeflow?start=17&length=25)

We should make this >= 1.1.0 and < 1.2.0 no? Because if we make a 1.1.* change, we should support it with this client